src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -0,0 +1,300 @@
+﻿using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Session;
+using Microsoft.Xunit.Performance.Sdk;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+using static Microsoft.Xunit.Performance.Api.Native.Windows.Kernel32;
+using static Microsoft.Xunit.Performance.Api.XunitPerformanceLogger;
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    internal static class ETWProfiler
```

Comment:
@brianrob This is actually cleaner with less dependencies on the classes used by the runner and related projects. It also wraps the call to the xunit api.

---
In reply to: [100390285](https://github.com/Microsoft/xunit-performance/pull/183#discussion_r100390285) [](ancestors = 100390285)